examples/README.md

```diff
@@ -34,6 +34,6 @@ You can stop the Jupyter server by double-pressing `ctrl+c` in your terminal.
 
 ## Maintaining the examples
 
-To add a new example, or to edit an existing one, make your changes in the `.py` version
-of the example and then run the `convert_py_to_ipynb.py` script. This will convert all
-`.py` files with notebook blocks e.g. `# %%` to the corresponding `.ipynb` file.
+To add a new example, or to edit an existing one, make your changes in the `.py` file
+of the example. Jupytext will automatically render all python files with markdown comments
+when percentage blocks (`# %%` or `# %% [markdown]`) to mark cells of the rendered notebook.
```

Comment:
It will render the .py file as a notebook within the jupyter interface (the python file becomes clickable and opens a notebook looking page). I will improve the comment.